students/sbolsen/lesson01/break_me.py

```diff
@@ -0,0 +1,25 @@
+import argparse
+
+def NameError():
+    name = 123
+    print(place)
+
+def TypeError():
+    integer = 123
+    print((integer) + 'This is a string and should generate an error')
+
+def SyntaxError():
+    for i in range(1,20)
+        print(i)
+
+def AttributeError():
```

Comment:
please use `snake_case` for vars and functions per python style guide

Reply:
Ok noted, I'll use snake_case moving forward. Thanks!